lib/components/account/account_body.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:flutter/material.dart';
+import 'package:project_fomo/components/shared/page_header.dart';
+import 'package:project_fomo/services/auth_service.dart';
+import 'package:project_fomo/style.dart';
+import 'package:project_fomo/services/user_service.dart';
+import 'package:project_fomo/components/shared/loading_indicator.dart';
+import 'package:project_fomo/models/user.dart';
+import 'package:project_fomo/models/user_data.dart';
+import 'package:provider/provider.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AccountBody extends StatefulWidget {
```

Comment:
Ideally you should only be using the AccountPage stateless widget, with streambuilders as necessary to pull in data.  There is not a true need for a stateful widget for account.